### ISSUE TITLE
PNG download button not working?

### ISSUE THREAD

**Issue by gkovac42** [game-icons/icons#772]
PNG button doesn't seem to register clicks as it used to anymore. Am I doing something wrong?

**Comment by Delapouite** [comment]
Hi. Some adjustements have been made last week. Does the problem persist?

**Comment by gkovac42** [comment]
Yes, the problem is still present in a slightly different form - the hand indicator now shows on hover but clicking the PNG button doesn't start the download. Instead it just scrolls the page down by a small amount.

**Comment by aloisdg** [comment]
Works on my side (Firefox 86.0 Windows 10 currently).

 @gkovac42 what browser/OS are you using?

**Comment by gkovac42** [comment]
> Works on my side (Firefox 86.0 Windows 10 currently).
> 
> @gkovac42 what browser/OS are you using? Could you provide a gif showcasing the problem?

As I've said in the original post, it's Chrome. And it still isn't working. Firefox works just fine for me too...
